### ISSUE TITLE
@Use x At y inherit bug

### ISSUE THREAD

**Issue by tarreislam** [tarreislam/TeenyScript#2]
```
@Namespace Main/Etc


$yelp = Func($a = 25)
    Return $a
EndFunc

$MyAu3 = Func()
    ; Using a later set Alias should not work like this
    ConsoleWrite(Main/$yelp(10))
EndFunc()


$MyClass = Func(Class)
    @Use Main/Etc At Main


    $get = Func()
        Return Main/$yelp()
    EndFunc

EndFunc


$Main = Func()

    ConsoleWrite(Main/Etc/$MyClass().get() & @CRLF)

EndFunc()
```

The first set alias namespace are inherited by all the others, causing weird errors. Will go thru some logic later

**Comment by tarreislam** [comment]
More problems found during debug.

Problem: https://regex101.com/r/qY9mY5/1
"Solution": https://regex101.com/r/qY9mY5/2

**Comment by tarreislam** [comment]
Fixed in 2.0.0
